src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/CaseDataService.java

```diff
@@ -353,4 +353,19 @@ public Map<String, Object> getPayloadOffFinremCaseData(FinremCaseData data) {
         }
         return notificationRequestPayload;
     }
+
+    public boolean isLitigantRepresented(CaseDetails caseDetails, boolean isApplicant) {
+        String isRepresented = nullToEmpty(caseDetails.getData().get(getLitigantRepresentedKey(caseDetails, isApplicant)));
```

Comment:
Is nullToEmpty required?

Reply:
Legacy code but agreed, I'll do the refactor.